src/components/VideoPlayer/index.tsx

```diff
@@ -52,14 +70,18 @@ const VideoPlayer: React.FC<VideoPlayerProps> = ({
   enableAutoplay = false,
   customFields = {},
   isPlaythrough = false,
-  aspectRatio: overrideAspectRatio,
+  shrinkToFit = false,
+  viewportPercentage = 75,
 }) => {
   // migration from video component
   // will fallback to uuid if id is undefined with defaulting to falsy ''
   const targetId = id || uuid;
 
   const { playthrough = false, autoplay = false } = customFields;
   const videoRef = useRef(targetId);
+  const containerRef = useRef();
+  const [aspectRatio, setAspectRatio] = useState(9 / 16); // default 16:9
```

Comment:
This confused me at first, especially with the comment. The state variable is `aspectRatio` which imply we are setting a 9:16 ratio.
Could we add a small comment why it's the opposite way

Reply:
clarified